sdk/core/System.Net.ClientModel/src/Message/RequestBodyContent.cs

```diff
@@ -0,0 +1,175 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.IO;
+using System.Net.ClientModel.Internal;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace System.Net.ClientModel.Core
+{
+    public abstract class RequestBodyContent : IDisposable
```

Comment:
We need to remove IDisposable from this type before GA.

If we do not, all callers of RequestBodyContent.Create will be forces to handle the content in a using block and this will complicate implementations.